### ISSUE TITLE
Make mobile sidebar expandable

### ISSUE THREAD

**Issue by mrkldshv** [supabase/supabase#582]
# Feature request

## Is your feature request related to a problem? Please describe.

When there is a view with sidebar on mobile opened, it's hard to navigate and interact with content which is on the right side.
![Screen Shot 2021-01-22 at 14 00 53](https://user-images.githubusercontent.com/47859603/105494517-369f9580-5cbb-11eb-8600-9cd8a95495fd.png)

## Describe the solution you'd like

Probably, it would be good to make this sidebar expandable, so that when it is hidden user can click through main content.

**Comment by sethburtonhall** [comment]
Assign me and I will take a look at this one.

**Comment by MildTomato** [comment]
Hey folks

We have quite a few layout issues using Supabase on mobile 😦   and even with the side navigation collapsing a lot of the pages are not optimised for mobile screens - we'll try and rectify some of this as we go along.

For the sidebar nav - we've started a UI library for Supabase over at https://github.com/supabase/ui and we are beginning to use new components in the Supabase Dashboard UI already, so any new UI functionality will probably be done via our new component library.

We haven't started any side bar components yet, but perhaps we could make them collapsable and more mobile friendly 😃 

[I've opened up a new issue](https://github.com/supabase/ui/issues/97) for this in @supabase/ui - if anyone would like to help out that would be awesome!
